source/includes/code-examples/find-many/FindMany.cs

```diff
@@ -36,15 +38,18 @@ public static void Main(string[] args)
     public static List<Restaurant> FindMultipleRestaurantsBuilderSync()
     {
         // start-find-builders-sync
+        // Creates a filter for all documents the have a "cuisine" value of "Pizza"
         var filter = Builders<Restaurant>.Filter
             .Eq("cuisine", "Pizza");
 
+        // Finds all documents that match the filter
```

Comment:
S: doesn't matter too much, but in the Java comments I used "retrieves" for every find operation instead of "finds"

```suggestion
        // Retrieves all documents that match the filter
```